openapi/installinator.json

```diff
@@ -2317,7 +2317,12 @@
           "slots_written"
         ]
       },
-      "TypedUuidForMupdateKind": {
+      "MupdateUuid": {
+        "x-rust-type": {
+          "crate": "omicron-uuid-kinds",
+          "path": "omicron_uuid_kinds::MupdateUuid",
+          "version": "*"
+        },
```

Comment:
The types are much more reasonably named now as well (they use the same type alias as the one defined in uuid-kinds).